tests/pub-data-urls_top-level-content/OPS/page_001.xhtml

```diff
@@ -0,0 +1,13 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Data URL not opening as a Top-level Content Document</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+    <h1>Data URL not opening as a Top-level Content Document</h1>
+    <p id="text1">The following image is an SVG contained within a <code>data:</code> URL.</p>
+  	<p id="text2">Test passes if you can see the SVG below (instead of on its own page as though it were a Top-level Content Document).</p>
```

Comment:
```suggestion
    <p id="text2">Test passes if you can see the SVG below (instead of on its own page as though it were a top-level content document).</p>
```